app/schemas/tech.relaycorp.gateway.data.database.AppDatabase/1.json

```diff
@@ -119,12 +119,46 @@
           }
         ],
         "foreignKeys": []
+      },
+      {
+        "tableName": "ParcelAck",
+        "createSql": "CREATE TABLE IF NOT EXISTS `${TABLE_NAME}` (`recipientAddress` TEXT NOT NULL, `senderAddress` TEXT NOT NULL, `messageId` TEXT NOT NULL, PRIMARY KEY(`recipientAddress`, `senderAddress`, `messageId`))",
+        "fields": [
```

Comment:
This is missing the expiry date of the parcel, which is important here: Records with an expiry date in the past should be automatically removed.

Reply:
Can't we get it from the Parcel table by relationship?

Reply:
I don't think so: The parcel could've been delivered to a local endpoint and subsequently deleted, before we get to generate/send PCAs for the courier.